### PR TITLE
Create Change_Themes.py

### PR DESCRIPTION
**Pull request by ool3**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@ool3

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by tirkarthi** [comment]
As noted in the other issue code changes need an issue in https://bugs.python.org and a description of your proposal over why this is needed in stdlib instead of being a third party package in PyPI.

File: Lib/tkinter/Change_Themes.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import tkinter as tk
+
+# create window
+window = tk.Tk()
+window.geometry('500x500')
+window['bg'] = '#cce'
+window.resizable(height=False, width=False)
+window.wm_attributes('-alpha', 0.95)
```
Comment (ool3) [inline Lib/tkinter/Change_Themes.py:8]: Create and set the window. Start background, crate opacity and etc.

File: Lib/tkinter/Change_Themes.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+# funcs
+def white():
+    window['bg'] = '#000'
+    btn_black['bg'] = 'white'
+    btn_black['fg'] = 'black'
+    btn_white['bg'] = 'black'
+    btn_white['fg'] = 'white'
+    btn_white["relief"] = "solid"
```
Comment (ool3) [inline Lib/tkinter/Change_Themes.py:20]: Create White theme.

File: Lib/tkinter/Change_Themes.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+
+def black():
+    window['bg'] = 'white'
+    btn_black['bg'] = 'white'
+    btn_black['fg'] = 'black'
+    btn_white['bg'] = 'black'
+    btn_white['fg'] = 'white'
```
Comment (ool3) [inline Lib/tkinter/Change_Themes.py:28]: Create Black theme

File: Lib/tkinter/Change_Themes.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+btn_black.place(x=20, y=80, width=200, height=60)
+
+btn_white = tk.Button(text='White Theme', command=black)
+btn_white["bg"] = "grey"
+btn_white["border"] = "0"
+btn_white['font'] = 'Arial 13'
+btn_white['fg'] = '#eee'
+btn_white.place(x=280, y=80, width=200, height=60)
+
+window.mainloop()
```
Comment (ool3) [inline Lib/tkinter/Change_Themes.py:47]: Create buttons

**Review by ool3 — COMMENTED** [review]
I add some comments

**Comment by csabella** [comment]
@ool3, as per @tirkarthi's message, this would need an issue on bugs.python.org in order to move it forward.  Also, in order to accept a pull request, you would need to sign the CLA as per the message from the bot.  Thank you!

**Comment by csabella** [comment]
@ool3, without a signed CLA, we are unable to accept this contribution.  Please sign the CLA as per the above instructions or this PR will be closed.  Thank you.

**Comment by csabella** [comment]
Closing for no CLA.  If the original author signs the CLA, then this can be reopened.

**Pull request closed without merging**